packages/forma-36-react-components/src/components/List/ListItem/ListItem.css

```diff
@@ -6,6 +6,7 @@
   font-size: var(--font-size-m);
   font-family: var(--font-stack-primary);
   line-height: var(--line-height-default);
+  list-style-type: initial;
```

Comment:
Should it be `initial` or `none`? isn't `initial` basically same as default behaviour?

Reply:
Yes it just enforces browser default. The way I understand #549, the problem is that the list style is being overriden to `none` by the web app because of specificity?

Reply:
Ah, sorry if it wasn't too clear. I meant people are rewriting `list-style-type: none` many times in the web app. So we need to add an option to reset the default `ul` and `li` styles, maybe a `reset={true}` prop? where we remove the style-type, margin and padding?

Reply:
There's no point in using the 36 `List` components in these cases though? The point of the component is to provide consistently styled lists.

I think ideally, we should default to a specific list style, and perhaps allow the user to override it using a `listStyle` prop.

Reply:
All right I've changed to disc, which is also the browser default 😉

Reply:
Sorry, I think my comment is confusing everyone, what's in the ticket is what's needed. 👍 
'disc' or 'initial' will work, I'd vote for what mike said though, about `listStyle` prop.

As for the `reset` prop to remove padding and margin, I ran into cases in planner and experience components where I needed to reset the styles specifically for building navigation lists like "user-profile", "main navigation", "app-switcher", ..etc. So it may be a good option to consider later.

Reply:
Sounds good, such a prop makes sense, but I think it should go on `<List />` and not on `<ListItem />` component. If you agree I think we can merge this PR for now.